parts/linux/cloud-init/artifacts/components.json

```diff
@@ -152,7 +152,8 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/apiserver-network-proxy/agent:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v0.1.6-hotfix.20240116"
+        "v0.1.6-hotfix.20240116",
```

Comment:
when will we be able to remove v0.1.6?

Reply:
I don't have an exact date but perhaps once the rollout is complete we can remove the old image.